sirepo/package_data/static/js/jupyterhublogin.js

```diff
@@ -3,31 +3,8 @@
 var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
 
-SIREPO.app.controller('JupyterhubMigrateController', function(authState, jupyterhubloginService, requestSender, $scope) {
-    const self = this;
-    self.isLoading = true;
-    requestSender.sendRequest(
-        'redirectJupyterHub',
-        () => {self.isLoading = false;}
-    );
-    self.migrate = function(doMigration) {
-        jupyterhubloginService.doMigration(doMigration);
-    };
-});
-
-SIREPO.app.controller('NameConflictController', function(requestSender, jupyterhubloginService, $route, $scope) {
-    const self = this;
-
-    self.noMigration = function() {
-        jupyterhubloginService.doMigration(false);
-    };
-
-    self.logout = function() {
```

Comment:
We still need logout.

Reply:
This is part of NameConflictController which is no longer used.